### PR TITLE
Merge recent changes to production

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

File: docs/edge-logic/supported-directives.md [hunk 1]
```diff
@@ -193,7 +193,9 @@ Limits the rate of response transmission to a client, in bytes/sec. Valid values
 
 <span class="badge">standard</span>
 
-Sets the initial amount of traffic after which the further transmission of a response to a client will be rate limited. We limit the value to an integer in [1-8] followed by ‘m’.
+**Default**: `limit_rate_after 0;` <br/>
```
Comment (burtonquantil) [inline docs/edge-logic/supported-directives.md:196]: It is inconsistent to show a default of "0" when the following text indicates the values are restricted between 1 and 8.

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Can you double check the limit_rate_after description?

File: docs/edge-logic/supported-directives.md (hunk 1 above)
Comment (nibinqtl) [reply docs/edge-logic/supported-directives.md]: updated with the latest default of '4m'.

**Review by burtonquantil — DISMISSED** [review]
Looks good. OK to merge.

File: docs/edge-logic/supported-directives.md [hunk 2]
```diff
@@ -84,22 +84,23 @@ Sets the request variable to the given value after the authorization request com
 
 Stops processing the current set of ngx_http_rewrite_module directives. No change to the public version. 
 
-### `custom_log_field`
-<span class="badge dark">advanced</span> <span class="badge primary">CDN360 Proprietary</span>
+### [`custom_log_field`](https://docs.cdnetworks.com/cdn/docs/edge-logic/supported-directives#custom_log_field)
```
Comment (nibinqtl) [inline docs/edge-logic/supported-directives.md:87]: @marcfishelquantil please remove all these self-links. You are hard coding them to cdnetworks.com.
They will be automatically generated, as you can see from https://docs.quantil.com/cdn/docs/edge-logic/supported-directives#custom_log_field

Comment (marcfishelquantil) [reply docs/edge-logic/supported-directives.md:87]: With pleasure. Next upload will have all of these removed.

Comment (nibinqtl) [reply docs/edge-logic/supported-directives.md:87]: @marcfishelquantil , you still haven't done so.

**Review by burtonquantil — DISMISSED** [review]
Looks OK to me

File: docs/edge-logic/supported-directives.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/edge-logic/supported-directives.md]: How about rewording the one sentence to the following, and then providing an example:

"We limit the value to an integer from [1-8] followed by ‘m’. For example:

**Review by snikitin-qtl — APPROVED** [review]
look good to me
